$NetBSD: patch-ac,v 1.2 1998/11/02 21:18:14 agc Exp $

We install the sfio header files into their own directory, to protect
any casual compilation which includes the standard include/ directory,
and so update the documentation to reflect this.

--- man/man3/sfio.3	1998/11/02 14:29:31	1.1
+++ man/man3/sfio.3	1998/11/02 14:29:57
@@ -16,6 +16,7 @@
 .nf
 .ft 5
 #include   <sfio.h>
+cc -I@prefix@/include/sfio ... -L@prefix@/lib -lsfio
 .ft 1
 .fi
 .Ss "DATA TYPES"
@@ -198,7 +199,7 @@
 .nf
 .ft 5
 #include   <stdio.h>
-cc ... -lstdio -lsfio
+cc -I@prefix@/include/sfio ... -L@prefix@/lib -lstdio -lsfio
 .ft 1
 .fi
 .SH DESCRIPTION
